packages/client/src/App.tsx

```diff
@@ -45,7 +62,7 @@ export const FeedScreen = ({ navigation } : Props) => {
   return (
     <ApolloProvider client={client}>
       <View style={styles.container}>
-        <MapView showRadius />
+        {/* <MapView showRadius /> */}
```

Comment:
Why is this commented out?

Reply:
did not mean to push these changes, they were for debugging, ill fix this